TWLight/users/templates/users/filter_section.html

```diff
@@ -24,6 +24,10 @@
           {% comment %}Translators: On the MyLibrary page (https://wikipedialibrary.wmflabs.org/users/my_library), this appears below introductory text and links to The Wikipedia Library Talk page on meta (https://meta.wikimedia.org/wiki/Talk:The_Wikipedia_Library) {% endcomment %}
           {% trans "Discuss" %}
         </a>
+        &ndash;
+        <a href="{{ user.userprofile.talk_page_i18n }}" class="twl-links twl-links-underline">
+          {{ user.userprofile.lang }}.wiki
```

Comment:
I find it confusing that the link can display a language-edition wiki when it can take a user to en wiki. I suggest setting it to talk_page_i18n so that the label matches the destination.

Reply:
Done!